src/ripple/server/Port.h

```diff
@@ -58,6 +58,9 @@ struct Port
     // port in the range [0, 65535] where 0 means unlimited.
     int limit = 0;
 
+    // Websocket disconnects if send queue exceeds this limit
+    std::uint16_t ws_queue_limit = 0;
```

Comment:
I know this will get set when parsing, but is it worth setting to the same default value of `100`?